.docker/redmine/zeitwerk.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+lib = Rails.root.join('lib/redmine')
+Rails.autoloaders.main.push_dir lib, namespace: Redmine
+Rails.application.config.watchable_dirs[lib.to_s] = [:rb] # patched by Renuo
```

Comment:
converting `lib` to a string fixes the following error
![CleanShot 2023-04-05 at 09 28 19](https://user-images.githubusercontent.com/39113171/230011463-4d996d16-af8f-4181-b8a5-7207ea2c550c.png)

`lib` is an instance of `Pathname` and thus doesn't support `gsub`. https://github.com/rails/rails/blob/6-1-stable/activesupport/lib/active_support/file_update_checker.rb#L42 strictly assumes the `dirs` key to be a string for my understanding

=> probably it would make sense to update https://github.com/ruby/gem_rbs_collection/blob/main/gems/activesupport/6.0/activesupport-generated.rbs#L7693